src/EnergyPlus/ChillerIndirectAbsorption.cc

```diff
@@ -106,7 +106,7 @@ namespace ChillerIndirectAbsorption {
     // OTHER NOTES:
     // Manufacturers performance data can be used to generate the coefficients for the model.
 
-    auto constexpr calcChillerAbsorptionIndirect("CALC Chiller:Absorption:Indirect ");
+    const char * calcChillerAbsorptionIndirect("CALC Chiller:Absorption:Indirect ");
```

Comment:
I think this is preferred, though it doesn't always work.

Reply:
I think it works as long as you don't end up trying to add another char * to the array.  As long as you are using it as a standalone variable, or passed into another function, or temporarily cast it to a string then it's fine.  But it should be noted that this could quietly cause a performance degradation if it is frequently passed into a function that expects a string argument.  In that case the string constructor will be called to create a new temporary based on this char array every time the function is called.